### ISSUE TITLE
@types/node crypto.generateKeyPairSync

### ISSUE THREAD

**Issue by sahanatroam** [DefinitelyTyped/DefinitelyTyped#33788]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/node:10.12.30` package and had problems.

```
const { publicKey, privateKey } = generateKeyPairSync('rsa', {
      modulusLength: 4096,  // the length of your key in bits
      publicKeyEncoding: {
        type: 'spki',       // recommended to be 'spki' by the Node.js docs
        format: 'pem'
      },
      privateKeyEncoding: {
        type: 'pkcs8',      // recommended to be 'pkcs8' by the Node.js docs
        format: 'pem',
      }
    });
```

When creating a key pair the `cipher` and `passphrase` should be optional, Will need a fix for this please for node 10.x

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1eeccafd4e15cf617660ceb4a57823b70c4be36b/types/node/crypto.d.ts#L401

**Comment by CupCakeArmy** [comment]
@sahanatroam Can you do a PR? This need fixing :)

**Comment by sahanatroam** [comment]
@CupCakeArmy happy to PR just need to understand how to create this properly, Will look in to this

**Comment by CupCakeArmy** [comment]
https://github.com/nodejs/help/issues/1839
I talked here with a dev. Basically you a re right, the `cipher` and `passphrase` should be made optional.

**Comment by jmorecroft** [comment]
Hi folks. I see this fix is in the latest version (@types/node@12.0.7) but not typings for earlier versions of Node. eg. @types/node@10.14.8 doesn't have this fix. Are we able to get it here also?
